### PR TITLE
Expose AC Loss and Low Battery states

### PR DESCRIPTION
**Pull request by austinmroczek**
This will allow us to send AC Loss and Low Battery to Home Assistant.

### CODE REVIEW

**Comment by uchagani** [comment]
Thanks for this.  I'll test this out in a couple days and merge it in.

**Comment by uchagani** [comment]
@austinmroczek can you rebase this against master please?

**Comment by austinmroczek** [comment]
This is my first rebase so please check my work closely.

File: total_connect_client/TotalConnectClient.py [hunk 1]
```diff
@@ -186,7 +189,7 @@ def get_location_by_location_name(self, location_name=False):
 
     def get_panel_meta_data(self, location_name=False):
         location = self.get_location_by_location_name(location_name)
-        
+
```
Comment (uchagani) [inline total_connect_client/TotalConnectClient.py]: remove empty line

File: total_connect_client/TotalConnectClient.py [hunk 2]
```diff
@@ -196,16 +199,45 @@ def get_panel_meta_data(self, location_name=False):
         if response.ResultCode != self.SUCCESS:
             raise Exception('Could not retrieve panel meta data')
 
+        self._panel_meta_data = zeep.helpers.serialize_object(response)
+
```
Comment (uchagani) [inline total_connect_client/TotalConnectClient.py]: another empty line

File: total_connect_client/test.py [hunk 3]
```diff
@@ -11,13 +11,24 @@
...
+  print('--- all location/system data ---')
   tc = TotalConnectClient.TotalConnectClient(sys.argv[1], sys.argv[2])
   location = tc.get_location_by_location_name()
-#  pprint(location)
+  pprint(location)
+
+  print('\n\n\n')
+
+  print('--- panel meta data ---')
+  tc = TotalConnectClient.TotalConnectClient(sys.argv[1], sys.argv[2])
```
Comment (uchagani) [inline total_connect_client/test.py]: we don't need another connection here.  let's get rid of it and use the `tc` object created above.

**Comment by uchagani** [comment]
@austinmroczek tested this and it works great.  if you can do the cleanups i recommended I'll go ahead and merge it in.  thanks!

**Comment by austinmroczek** [comment]
Fixes made.  Thanks

**Pull request merged**